UnityProject/Assets/Scripts/Messages/Server/SetActionUIMessage.cs

```diff
@@ -87,6 +89,20 @@ private void ProcessActionGUI(NetMessage msg)
 					action = spellData.AddToPlayer(NetworkObject.GetComponent<Mind>());
 				}
 			}
+			else if (msg.changelingAbilityListIndex >= 0)
```

Comment:
what stops this from being a normal UI action?

Reply:
That normal UI action doesn`t have index for call to ability